packages/checkout/widgets-lib/src/lib/walletConnect.ts

```diff
@@ -2,6 +2,7 @@ import { WalletConnectModal } from '@walletconnect/modal';
 import EthereumProvider from '@walletconnect/ethereum-provider';
 import { ChainId, WidgetTheme } from '@imtbl/checkout-sdk';
 import { Environment } from '@imtbl/config';
+import { ConnectConfig } from '@imtbl/checkout-sdk/dist/types';
```

Comment:
This hasn't been exported properly from the sdk. Either we export the type from there or create an equivalent type in widgets lib

Reply:
Oh good catch! I've just used a local type for now, the only property I'm interested on that is `walletconnect` anyway so should be pretty safe if they go out of sync.